docs/storage.md

```diff
@@ -316,6 +322,39 @@ Now you can create the SR where `server` is your mon ip.
 * Do not use admin keyring for production, but make a separate key with only necessary privileges https://docs.ceph.com/en/latest/rados/operations/user-management/
 :::
 
+### MooseFS
+
+Shared, thin-provisioned storage. Available since XCP-ng 8.2.
+
+MooseFS is a fault-tolerant, highly available, highly performing, scaling-out, network distributed file system.  It is POSIX compliant and acts like any other Unix-like file system.
+SR driver was contributed directly by MooseFS Developement Team.
+
+:::warning
+The MooseFS client is not included with XCP-ng, so it must be installed on dom0 from the official MooseFS repository.
+:::
+
+Installation steps
+```
+curl "https://ppa.moosefs.com/RPM-GPG-KEY-MooseFS" > /etc/pki/rpm-gpg/RPM-GPG-KEY-MooseFS
+curl "http://ppa.moosefs.com/MooseFS-3-el7.repo" > /etc/yum.repos.d/MooseFS.repo
+yum install moosefs-client
+```
+:::tip
+By default moosefs plugin is not enabled on whitelist of SM plugins in /etc/xapi.conf so we have to add it on `sm-plugins` section.
+:::
+
+Now when MooseFS client is installed you can connect existing [MooseFS cluster](https://moosefs.com/support/#documentation) and create shared SR for all hosts in the pool.
```

Comment:
I'm not a native english speaker, but I think "you can connect existing MooseFS cluster" should be "you can connect **to an** existing MooseFS cluster"?